packages/kit/src/runtime/client/client.js

```diff
@@ -189,12 +189,6 @@ let target;
 /** @type {import('./types.js').SvelteKitApp} */
 export let app;
 
-/** @type {Record<string, any>} */
-// we have to conditionally access the properties of `__SVELTEKIT_PAYLOAD__`
-// because it will be `undefined` when users import the exports from this module.
-// It's only defined when the server renders a page.
-export const remote_responses = __SVELTEKIT_PAYLOAD__?.data ?? {};
```

Comment:
We could make this `export let` and assign the value inside `start` - basically what we do for `export let app`, too. That way you don't have to access globals in the other places and it minifies a tiny bit better.

Reply:
good idea, made that change